bench/src/main/scala/tsec/CipherBench.scala

```diff
@@ -55,19 +56,19 @@ class CipherBench {
     * to view the related overhead, but we do not care about sequencing them
     */
   @Benchmark
-  def testTSecJCA(): Unit =
+  def testTSecJCA(): Any =
     jcaAES
       .encrypt(longPlaintext, jcaAESKey)
       .unsafeRunSync()
 
   @Benchmark
-  def testLibSodiumAES(): Unit =
+  def testLibSodiumAES(): Any =
     AES256GCM
       .encrypt[IO](nPlaintext, sodiumAESKey)
       .unsafeRunSync()
 
   @Benchmark
-  def testLibSodiumXChacha20(): Unit =
+  def testLibSodiumXChacha20(): Any =
```

Comment:
Why the `Any` - Value discard?  If so `.encrypt(...).void.unsafeRunSync` : `_=> ()` should be manageable latency

Reply:
@ChristopherDavenport I would prefer not to introduce the noise of one more `map`; these benchmarks are going to be a little messy either way.

@jmcardon [if you return a value, it will `bh.consume` it](http://tutorials.jenkov.com/java-performance/jmh.html#return-value-from-benchmark-method).

Reply:
ha this is good to know :+1: